scripts/ci-e2e.sh

```diff
@@ -54,7 +54,7 @@ kind:prepullAdditionalImages
 # Configure e2e tests
 export GINKGO_NODES=3
 export GINKGO_NOCOLOR=true
-export GINKGO_ARGS="${GINKGO_ARGS:-"--fail-fast"}"
+export GINKGO_ARGS="${GINKGO_ARGS:-"--fail-fast=false"}"
```

Comment:
```suggestion
export GINKGO_ARGS="${GINKGO_ARGS:-""}"
```

What about just not setting the field? (afaik not setting is equal to setting it to false)

Reply:
Sorry just saw your comment afterwards. I would prefer not setting it. Just feels cleaner if we are setting it to its default value (there are many other fields that we're also not setting with their default value)